Fix "error: constant expression evaluates to 212 which cannot be narrowed to
type 'char'"
--- pcapFileHandle.h.orig	2007-07-05 11:12:20.000000000 -0500
+++ pcapFileHandle.h	2020-08-20 04:51:37.000000000 -0500
@@ -16,7 +16,12 @@
 
 #define PCAP_HEADER_SIZE 24
 
-const char pcap_header[] = { 0xd4,0xc3,0xb2,0xa1,0x02,0x00,0x04,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x20,0x4e,0x00,0x00,0x01,0x00,0x00,0x00 };
+const char pcap_header[] = {
+	(char)0xd4,(char)0xc3,(char)0xb2,(char)0xa1,(char)0x02,(char)0x00,
+	(char)0x04,(char)0x00,(char)0x00,(char)0x00,(char)0x00,(char)0x00,
+	(char)0x00,(char)0x00,(char)0x00,(char)0x00,(char)0x20,(char)0x4e,
+	(char)0x00,(char)0x00,(char)0x01,(char)0x00,(char)0x00,(char)0x00
+};
 
 class pcapFileHandle : public fileHandle {
 
